src/main/java/Duke.java

```diff
@@ -1,23 +1,31 @@
 import java.util.Scanner;
 
 public class Duke {
-    //array to store task list
-    private static Task[] tasks = new Task[100];
+
+    /*ATTRIBUTES*/
+
+    private static Task[] tasks = new Task[100]; //array to store task list
+    private static int taskCount; //store total number of tasks
+
+
+    /*METHODS*/
 
     //adds task to the task list
-    public static void addTask(Task t, int count) {
+    public static void addTask(Task t) {
         printHorizontalLine();
-        tasks[count] = t;
+        tasks[taskCount] = t;
```

Comment:
I would recommend a different variable name as compared to just a letter t, so it is more easily relatable.